sofl/preferences.py

```
@@ -502,95 +523,111 @@ def setup_online_fix_settings(self) -> None:
+        self, combo: Adw.ComboRow, proton_versions: list[str], schema_key: str
+    ) -> None:
         """Setup Proton version selection combo box"""
         # Create model for combo box
         proton_model = Gtk.StringList.new([version for version in proton_versions])
         combo.set_model(proton_model)
-        
+
         # Get current selection from settings
         try:
             current_proton = shared.schema.get_string(schema_key)
         except GLib.Error as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)